astroquery/xmm_newton/core.py

```diff
@@ -137,36 +109,18 @@ def download_data(self, observation_id, filename=None, verbose=False,
         if verbose:
             log.info(link)
 
-        response = None
-        try:
-            response = self._request('GET', link, save=True, cache=True)
-        except HTTPError:
-            log.error("No results found in URL {0}".format(link))
+        response = self._request('GET', link, save=True, cache=True)
 
-        if response:
-            if filename is None:
-                filename = observation_id + ".tar"
-
-            log.info("Copying file to {0}...".format(filename))
-            path = os.getcwd()
-            destfile = os.path.join(path)
-            file = None
-            try:
-                os.makedirs(destfile, exist_ok=True)
-                file = open(os.path.join(destfile, filename), "w+")
-            except OSError:
-                log.error("Creation of the directory %s failed" % destfile)
-            else:
-                if verbose:
-                    log.info("Successfully created the directory %s " %
-                             destfile)
-                shutil.copy(response,
-                            os.path.join(path, filename))
+        if filename is None:
+            filename = observation_id + ".tar"
 
-            if verbose:
-                log.info("Wrote {0} to {1}".format(link, filename))
+        log.info("Copying file to {0}...".format(filename))
+        path = os.getcwd()
+        shutil.copy(response,
+                    os.path.join(path, filename))
```

Comment:
This is fine, but why not use the much simpler `"."` instead of `os.getcwd()` ?  Or even simpler, just this:
```suggestion
        shutil.copy(response, filename)
```
if your goal is to copy into the current working directory, you don't need any of this extra stuff around.